src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements QuadrangularArea, TriangleArea {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `draw()` and `getArea()` methods are not conceptually close to each other, so they should not be in the same interface.